portfolio/src/main/webapp/script.js

```diff
@@ -16,12 +16,20 @@
  * Adds a random greeting to the page.
```

Comment:
JS files should have a file level comment explaining the file details. Reach out to @shradha-khapra to know how it can be mentioned.